docs/infrastructure-metadata-service/index.md

```diff
@@ -0,0 +1,26 @@
+# Infrastructure Metadata Service
+
+## About Metadata Service
+
+Several cloud providers support a [metadata service](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html) for their instances. The metadata service allows users to access their instance metadata. Instance metadata is any data associated with the instance such as hostname, security groups, user data etc. that can be used to configure or manage the instance.
+
+## Securing access to Metadata Service
+
+If your cloud provider runs a metadata service, you would typically want pods created in 
+OpenShift to not be able to access the metadata service. 
+OpenShift's native CNI plugins (OpenShift SDN and OVN-Kubernetes) provide a way to secure
+your metadata service by adding IP Tables rules to block access. 
+
+### OpenShift SDN
+
+To block access to the metadata service from pods in OpenShift, please add your
+metadata service IP based on your platform to the [generateIPTablesRules](https://github.com/openshift/sdn/blob/master/pkg/cmd/openshift-sdn-cni/openshift-sdn.go#L121) function.
+
+### OVN Kubernetes
+
+To block access to the metadata service from pods in OpenShift, please add your
+metadata service IP based on your platform to the [setupIPTablesBlocks](https://github.com/openshift/ovn-kubernetes/blob/master/go-controller/pkg/cni/OCP_HACKS.go#L35) function.
+
+### Questions
+
+Questions can be directed to the OpenShift Slack channel **#forum-sdn**
```

Comment:
Is this channel accessible by users outside Red Hat?

Reply:
No, I assume we would have guest/partner slack accounts for infra provider engineers working with OCP Engineering?

Reply:
In the past, I've seen us setup dedicated Slack channels that partner's can join as a "single channel guest". But I'm not aware of partner's being able to join channels like #forum-sdn.

Reply:
we might just want to mention reaching out to their red hat partner for questions. it seems some like of the relationships result in partner slack channels, and others do not.